### PR TITLE
[Docs] Translate tutorials/evaluation.md

### PR DESCRIPTION
**Pull request by LEFTeyex**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/tutorials/evaluation.md [hunk 1]
```diff
@@ -1,3 +1,138 @@
 # Evaluation
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/evaluation.html).
+In model validation and model test, it is often necessary to make a quantitative evaluation of model accuracy. We can do this by specifying the Metric in the configuration file.
```
Comment (HAOCHENYE) [inline docs/en/tutorials/evaluation.md:3]:
```suggestion
In model validation and model test, it is often necessary to make a quantitative evaluation of model accuracy. We can do this by specifying the metrics(type and arguments of `Metric`) in the configuration file.
```

File: docs/en/tutorials/evaluation.md [hunk 2]
```diff
@@ -1,3 +1,138 @@
...
+
+## Evaluation in model training or test
+
+### Using a single evaluation metric
+
+When training or testing a model based on MMEngine, users only need to specify the evaluation metrics for the validation and test stages in the configuration file via the `val_evaluator` and `test_evaluator` fields, respectively. For example, when training a classification model using [MMClassification](https://github.com/open-mmlab/mmclassification), you want to evaluate the top-1 and top-5 classification accuracy during the validation stage of the model, you can configure it as follows:
```
Comment (HAOCHENYE) [inline docs/en/tutorials/evaluation.md:9]:
```suggestion
When training or testing a model based on MMEngine, users only need to specify the evaluation metrics for the validation and test stages in the configuration file via the `val_evaluator` and `test_evaluator` fields, respectively. For example, when training a classification model using [MMClassification](https://github.com/open-mmlab/mmclassification), you want to evaluate the top-1 and top-5 classification accuracy during the validation stage of the model,  and then you can configure it as follows:
```

File: docs/en/tutorials/evaluation.md [hunk 3]
```diff
@@ -1,3 +1,138 @@
...
+        seg_prefix='annotations/panoptic_val2017',
+    )
+]
+```
+
+### Custom evaluation metrics
+
+If the common evaluation metrics provided in the algorithm library cannot meet the needs, users can also add custom evaluation metrics. As an example, we present the implementation of custom metrics with the simplified classification accuracy:
+
+1. When defining a new evaluation metric class, you need to inherit the base class [BaseMetric](../../../mmengine/evaluator/metric.py) (for the introduction of this base class, you can refer to the [design document](../design/evaluation.md)). In addition, the evaluation metric class needs to be registered with the registrar `METRICS` (for a description of the registrar, please refer to the [Registry documentation](../advanced_tutorials/registry.md)).
```
Comment (HAOCHENYE) [inline docs/en/tutorials/evaluation.md:42]: The URL will be generated by MyST during building the docs automatically.
```suggestion
1. When defining a new evaluation metric class, you need to inherit the base class [BaseMetric](mmengine.evaluator.BaseMetric) (for the introduction of this base class, you can refer to the [design document](../design/evaluation.md)). In addition, the evaluation metric class needs to be registered with the registrar `METRICS` (for a description of the registrar, please refer to the [Registry documentation](../advanced_tutorials/registry.md)).
```

**Review by HAOCHENYE — COMMENTED** [review]
Thaaaaanks for your contribution!

File: docs/en/tutorials/evaluation.md [hunk 4]
```diff
@@ -1,3 +1,138 @@
...
+]
+```
+
+### Custom evaluation metrics
+
+If the common evaluation metrics provided in the algorithm library cannot meet the needs, users can also add custom evaluation metrics. As an example, we present the implementation of custom metrics with the simplified classification accuracy:
+
+1. When defining a new evaluation metric class, you need to inherit the base class [BaseMetric](mmengine.evaluator.BaseMetric) (for the introduction of this base class, you can refer to the [design document](../design/evaluation.md)). In addition, the evaluation metric class needs to be registered with the registrar `METRICS` (for a description of the registrar, please refer to the [Registry documentation](../advanced_tutorials/registry.md)).
+
+2. Implement the `process()` method. This method has two input parameters, which are the test data samples of a batch `data_batch` and the model prediction results `data_samples`. We take the sample category labels and the classification prediction results from them and store them in `self.results` respectively.
```
Comment (zhouzaida) [inline docs/en/tutorials/evaluation.md:44]:
```suggestion
2. Implement the `process()` method. This method has two input parameters, which are a batch of test data samples, `data_batch`, and model prediction results, `data_samples`. We extract the sample category labels and the classification prediction results from them and store them in `self.results` respectively.
```

File: docs/en/tutorials/evaluation.md [hunk 5]
```diff
@@ -1,3 +1,138 @@
...
+
+3. Implement the `compute_metrics()` method. This method has one input parameter `results`, which holds the results of all batches of test data processed by the `process()` method. The sample category labels and classification predictions are extracted from the results to calculate the classification accuracy `acc`. Finally, the calculated evaluation metrics are returned in the form of a dictionary.
```
Comment (zhouzaida) [inline docs/en/tutorials/evaluation.md:46]:
```suggestion
3. Implement the `compute_metrics()` method. This method has one input parameter `results`, which holds the results of all batches of test data processed by the `process()` method. The sample category labels and classification predictions are extracted from the results to calculate the classification accuracy (`acc`). Finally, the calculated evaluation metrics are returned in the form of a dictionary.
```

File: docs/en/tutorials/evaluation.md [hunk 6]
```diff
@@ -1,3 +1,138 @@
...
+evaluator = Evaluator(metrics=dict(type='Accuracy', top_k=(1, 5)))
+
+# Reads the test data from a file. The data format needs to refer to the metric used.
+data = load('test_data.pkl')
+
+# The model prediction result is read from the file. The result is inferred by the algorithm to be evaluated on the test dataset.
+# The data format needs to refer to the metric used.
+data_samples = load('prediction.pkl')
+
+# Call the evaluator offline review interface and get the evaluation results
```
Comment (zhouzaida) [inline docs/en/tutorials/evaluation.md:134]:
```suggestion
# Call the evaluator offline evaluation interface and get the evaluation results
```

File: docs/en/tutorials/evaluation.md [hunk 7]
```diff
@@ -1,3 +1,138 @@
...
+# chunk_size indicates the number of samples processed at a time, which can be adjusted according to the memory size
+results = evaluator.offline_evaluate(data, data_samples, chunk_size=128)
+
```
Comment (zhouzaida) [inline docs/en/tutorials/evaluation.md:137]:
```suggestion
```

File: docs/zh_cn/design/evaluation.md [hunk 8]
```diff
@@ -4,7 +4,7 @@
 
 在模型验证和模型测试中，通常需要对模型精度做定量评测。在 MMEngine 中实现了评测指标（Metric）和评测器（Evaluator）来完成这一功能。
 
-- **评测指标** 用于根据测试数据和模型预测结果，完成特定模型精度指标的计算。在 OpenMMLab 各算法库中提供了对应任务的常用评测指标，如 [MMClassification](https://github.com/open-mmlab/mmclassification) 中提供了[Accuracy](https://mmclassification.readthedocs.io/en/1.x/api/generated/mmcls.evaluation.Accuracy.html#mmcls.evaluation.Accuracy) 用于计算分类模型的 Top-k 分类正确率；MMDetection 中提供了 [COCOMetric](https://github.com/open-mmlab/mmdetection/blob/3.x/mmdet/evaluation/metrics/coco_metric.py) 用于计算目标检测模型的 AP，AR 等评测指标。评测指标与数据集解耦，如 COCOMetric 也可用于 COCO 以外的目标检测数据集上。
+- **评测指标** 用于根据测试数据和模型预测结果，完成特定模型精度指标的计算。在 OpenMMLab 各算法库中提供了对应任务的常用评测指标，如 [MMClassification](https://github.com/open-mmlab/mmclassification) 中提供了[Accuracy](https://mmclassification.readthedocs.io/en/1.x/api/generated/mmcls.evaluation.Accuracy.html#mmcls.evaluation.Accuracy) 用于计算分类模型的 Top-k 分类正确率；[MMDetection](https://github.com/open-mmlab/mmdetection) 中提供了 [COCOMetric](https://github.com/open-mmlab/mmdetection/blob/3.x/mmdet/evaluation/metrics/coco_metric.py) 用于计算目标检测模型的 AP，AR 等评测指标。评测指标与数据集解耦，如 COCOMetric 也可用于 COCO 以外的目标检测数据集上。
```
Comment (zhouzaida) [inline docs/zh_cn/design/evaluation.md:7]:
```suggestion
- **评测指标** 用于根据测试数据和模型预测结果，完成特定模型精度指标的计算。在 OpenMMLab 各算法库中提供了对应任务的常用评测指标，如 [MMClassification](https://github.com/open-mmlab/mmclassification) 中提供了[Accuracy](https://mmpretrain.readthedocs.io/en/latest/api/generated/mmpretrain.evaluation.Accuracy.html#mmpretrain.evaluation.Accuracy) 用于计算分类模型的 Top-k 分类正确率；[MMDetection](https://github.com/open-mmlab/mmdetection) 中提供了 [COCOMetric](https://github.com/open-mmlab/mmdetection/blob/main/mmdet/evaluation/metrics/coco_metric.py) 用于计算目标检测模型的 AP，AR 等评测指标。评测指标与数据集解耦，如 COCOMetric 也可用于 COCO 以外的目标检测数据集上。
```

**Review by zhouzaida — DISMISSED** [review]

File: docs/en/tutorials/evaluation.md [hunk 9]
```diff
@@ -1,3 +1,137 @@
...
+In model validation and testing, it is often necessary to make a quantitative evaluation of model accuracy. We can achieve this by specifying the metrics in the configuration file.
+
+## Evaluation in model training or testing
+
+### Using a single evaluation metric
+
+When training or testing a model based on MMEngine, users only need to specify the evaluation metrics for the validation and testing stages through the `val_evaluator` and `test_evaluator` fields in the configuration file. For example, when using [MMClassification](https://github.com/open-mmlab/mmclassification) to train a classification model, if the user wants to evaluate the top-1 and top-5 classification accuracy during the model validation stage, they can configure it as follows:
+
+```python
+val_evaluator = dict(type='Accuracy', top_k=(1, 5))  # using classification accuracy evaluation metric
```
Comment (zhouzaida) [inline docs/en/tutorials/evaluation.md:12]:
```suggestion
# using classification accuracy evaluation metric
val_evaluator = dict(type='Accuracy', top_k=(1, 5))
```

**Pull request merged**